accessible_api/AccessibilityRequestHandler.py

```diff
@@ -0,0 +1,124 @@
+""" Service requests for accessible objects """
+
+import sys
+from BaseHTTPServer import BaseHTTPRequestHandler
+from re import search
+from urlparse import urlsplit, parse_qsl
+from json import dumps
+from comtypes import CoInitialize
+from accessible_lib.scripts.accessible import accessible
+from accessible_lib.scripts.event import event
+from accessible_lib.scripts.commands import execute_command
+
+
+class AccessibilityRequestHandler(BaseHTTPRequestHandler):
+    def _successful_response(self, json):
+        self.send_response(200)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+        self.wfile.write(json)
+
+    def _bad_response(self, message):
+        self.send_response(400, message)
+        self.send_header('Content-Type', 'application/json')
+        self.end_headers()
+
+    def _set_identifiers(self, params):
+        name = params.get('name')
+        role = params.get('role')
+
+        identifiers = {}
+        if name is not None:
+            identifiers["Name"] = name
+        if role is not None:
+            identifiers["Role"] = role
+
+        return identifiers
+
+    def do_GET(self):
+        if search('/accessible', self.path) is not None:
+            params = dict(parse_qsl(urlsplit(self.path).query))
+
+            _interface = params.get('interface')
+            _identifiers = self._set_identifiers(params)
+            _depth = params.get('depth')
+            _acc_obj = accessible(_interface, _identifiers)
+
+            if _acc_obj.found:
+                _json = _acc_obj.serialize(_depth)
+                self._successful_response(dumps({_interface: _json}))
+            else:
+                self._bad_response('Bad Request: Accessible does not exist')
+
+        elif search('/event', self.path) is not None:
+            params = dict(parse_qsl(urlsplit(self.path).query))
+            _interface = params.get('interface')
+            _identifiers = self._set_identifiers(params)
+            _event = params.get('type')
```

Comment:
misnomer event -> eventType, but see if parsing it like that is needed at all with comment above.